packages/ckeditor5-dev-tests/lib/utils/automated-tests/getkarmaconfig.js

```diff
@@ -179,6 +179,17 @@ module.exports = function getKarmaConfig( options ) {
 		// Otherwise, Karma compiles sources once, and the test bundle uses old code.
 		// For the future reference: https://youtrack.jetbrains.com/issue/WEB-12496.
 		karmaConfig.webpack.watch = true;
+
+		// Remove istanbul-instrumenter if coverage reporter was removed by overrides
+		// (especially when debugging in Intellij IDE).
+		if ( !karmaConfig.reporters.includes( 'coverage' ) && karmaConfig.webpack.module ) {
+			const moduleRules = karmaConfig.webpack.module.rules;
+			const ruleIdx = moduleRules.findIndex( rule => rule.loader == 'istanbul-instrumenter-loader' );
+
+			if ( ruleIdx != -1 ) {
+				moduleRules.splice( ruleIdx, 1 );
+			}
+		}
```

Comment:
This line deserves at least a single check to avoid accidental removal in the future.

Reply:
Added test for this branch and for `options.coverage`.